azure/image/misc-settings.sh

```diff
@@ -131,6 +135,13 @@ TLS_OPTIONS=-cert-file /etc/certificates/tls.crt -cert-key /etc/certificates/tls
 END
 fi
 
+mkdir -p /etc/systemd/system/attestation-agent.service.d
+cat <<END >/etc/systemd/system/attestation-agent.service.d/10-grpc.conf
+[Service]
+ExecStart=
+ExecStart=/usr/local/bin/attestation-agent.grpc --getresource_sock 127.0.0.1:50001
+END
+
```

Comment:
So this will override https://github.com/confidential-containers/cloud-api-adaptor/blob/main/podvm/files/etc/systemd/system/attestation-agent.service.  Do we want this ? ( I have no preference, hence checking)

Reply:
There is an discussion of this regard under this comment https://github.com/confidential-containers/cloud-api-adaptor/pull/1202#discussion_r1264939958 :wink: